src/blas/CUBLAS.jl

```diff
@@ -35,6 +37,24 @@ function handle()
     return _handle[]
 end
 
+function xt_handle()
+    if _xt_handle[] == C_NULL
+        @assert isassigned(active_context) # some other call should have initialized CUDA
+        _xt_handle[] = get!(_xt_handles, active_context[]) do
+            context = active_context[]
+            dev = CUDAdrv.device(context)
+            handle = cublasXtCreate()
+            cublasXtDeviceSelect(handle, 1, [dev.handle])
```

Comment:
why not default to using all available devices? isn't that the whole point of cublasXt?
it seems like you can't switch modes afterwards (or at least not between calls), so it'd need to happen here.

>  Currently the device configuration is static and cannot be changed between Math function calls. In that regard, this function should be called only once after cublasXtCreate. To be able to run multiple configurations, multiple cublasXt API contexts should be created.

Reply:
I set this for testing purposes and forgot about it! I'll try adding in more devices too :)